src/pipeline/eval.py

```diff
@@ -44,11 +44,12 @@ async def _gen_eval_on_records(eval_prompts, eval_model, eval_workers):
     _gen_eval_on_records simultaneously generates evaluations on the eval_prompts
     """
     assessments = []
+    keys_to_check = ["similarity_assessment", "precision_assessment"]
```

Comment:
This should be configured or made a constant more appropriately. Also, why not make it a `set()` from the get-go?